src/components/resource/resource-show.js

```diff
@@ -438,13 +438,13 @@ class ResourceShow extends Component {
               </Accordion>
 
               {showCustomLables &&
-                <CustomLabelsAccordion
-                  id="resourceShowCustomLabels"
-                  isOpen={sections.resourceShowCustomLabels}
-                  onToggle={this.handleSectionToggle}
-                  section={CustomLabelsShowSection}
-                  userDefinedFields={userDefinedFields}
-                />}
+              <CustomLabelsAccordion
+                id="resourceShowCustomLabels"
+                isOpen={sections.resourceShowCustomLabels}
+                onToggle={this.handleSectionToggle}
+                section={CustomLabelsShowSection}
+                userDefinedFields={userDefinedFields}
+              />}
```

Comment:
Why has these spaces been removed? According to the code style, this expression is written correctly